data/CMakeLists.txt

```diff
@@ -18,5 +18,37 @@ if(INSTALL_TESTS AND OPENCV_TEST_DATA_PATH)
             DIRECTORY_PERMISSIONS OWNER_WRITE OWNER_READ OWNER_EXECUTE
               GROUP_READ GROUP_EXECUTE WORLD_READ WORLD_EXECUTE
             DESTINATION share/OpenCV/testdata COMPONENT tests)
+    if(BUILD_opencv_python)
```

Comment:
INSTALL_TESTS is already checked in line 12. Here I've add data necessary for python tests to already packaged test data